conf/custom_views.py

```
@@ -0,0 +1,18 @@
+from django.http import JsonResponse
+from rest_framework.generics import ListAPIView
+
+from conf.helpers import str_to_bool
+
+
+class OptionallyPaginatedEndpoint(ListAPIView):
```

Can this be PaginatedAPIView? I don't think its necessary to embed the the param name behaviour in the class name?

---

Well by default ListAPIView is just a PaginatedAPIView so it would be misleading to name it so. This is new functionality we're introducing to allow to to be optionally paginated